### ISSUE TITLE
invalid: unknown revision since 0.2.6

### ISSUE THREAD

**Issue by cloudlena** [livebud/bud#360]
Since `0.2.6`, I'm getting the following error message upon running `but create` and `but new`:

```shell
$ buds3 bud new controller todo
| /home/lena/projects/todos/go.mod:6:2: require github.com/livebud/bud: version "0.2.6" invalid: must be of the form v1.2.3
```

I think the `v` suffix is missing for the version number in the generated `go.mod` file.

**Comment by matthewmueller** [comment]
Thanks for the heads up! Just fixed this in #360 and added a test so this can't happen again.

**Comment by matthewmueller** [comment]
`0.2.7` is out!
